src/modules/72-triggers/pages/triggers/views/WebhookPipelineInputPanel.tsx

```diff
@@ -150,23 +148,12 @@ const WebhookPipelineInputPanelForm: React.FC<WebhookPipelineInputPanelPropsInte
   const { getString } = useStrings()
   const ciCodebaseBuildValue = formikProps.values?.pipeline?.properties?.ci?.codebase?.build
   let hasEverRendered = typeof ciCodebaseBuildValue === 'object' && !isEmpty(ciCodebaseBuildValue)
-  const [pipelineWithoutRefs, setPipelineWithoutRefs] = useState<PipelineInfoConfig>(originalPipeline)
-
-  useEffect(() => {
-    try {
-      if (resolvedTemplatesPipelineYaml) {
-        setPipelineWithoutRefs(parse(resolvedTemplatesPipelineYaml)?.pipeline)
-      }
-    } catch (e) {
-      // ignore error
-    }
-  }, [resolvedTemplatesPipelineYaml])
 
   useEffect(() => {
     const shouldInjectCloneCodebase =
-      !isEmpty(pipelineWithoutRefs?.properties?.ci?.codebase) &&
-      (pipelineWithoutRefs?.stages?.some((stage: any) => stage?.stage?.spec?.cloneCodebase) ||
-        pipelineWithoutRefs?.stages?.[0]?.parallel?.some((stage: any) => stage?.stage?.spec?.cloneCodebase))
+      !isEmpty(resolvedPipeline?.properties?.ci?.codebase) &&
+      (resolvedPipeline?.stages?.some((stage: any) => stage?.stage?.spec?.cloneCodebase) ||
+        resolvedPipeline?.stages?.[0]?.parallel?.some((stage: any) => stage?.stage?.spec?.cloneCodebase))
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9238322)
```suggestion
        resolvedPipeline?.stages?.[0]?.parallel?.some((stage: unknown) => stage?.stage?.spec?.cloneCodebase))
```